### ISSUE TITLE
OrderConfirmation not send when darique module is activated

### ISSUE THREAD

**Issue by AaronTajriba** [mollie/PrestaShop1.7#549]
### Expected behavior
The order confirmation email should be send

### Actual behavior
It does not send the order confirmation only the payment accepted

### Steps to reproduce the behavior
You need to have the Darique module enabled.

This issue was not here in previous versions of the mollie plugin (4+).

### Environment
* PrestaShop version: 1.7.6.8
* Mollie module version: 5.0.1
* PHP version: 7.3

**Comment by loonas** [comment]
Hi @AaronTajriba,

Due to inactivity, I will mark this issue as resolved. Let us know if you will have any other questions or when you would like to continue!

Best regards,
Invertus Support team

**Comment by AaronTajriba** [comment]
Hi,

sorry for the delay, i've send an email with our cridentials.

**Comment by Saimis777** [comment]
Hi @AaronTajriba,

We could not find the issue, but checked with other payment modules and it didn’t work on them too. Tried to install "darique" module on our shop and it worked fine. In the email logs we can see that email is sent but for some reason we are not getting it. Most likely it looks like its a theme issue or maybe mail configurations but it looks like Mollie has nothing to do with this issue.

Let us know if you have any other questions!

--
Best regards,
Invertus Support team

**Comment by AaronTajriba** [comment]
@Saimis777 

I've tested it again and the problem is ONLY  when i checkout with an mollie payment method (IDEAL, PAYPAL, MrCash/Bancontact , Credit Card or SOFORT Bankieren). When i send it with bankoverschrijving it all works fine. 

This issue appeared when we updated the mollie module.

**Comment by Saimis777** [comment]
Hey @AaronTajriba ,

Did you try to PHP mailer as well? Or is it only with the custom SMTP settings that you are running on your shop?

Looking forward to your reply!

--
Best regards,
Invertus Support team

**Comment by Saimis777** [comment]
Hi again @AaronTajriba ,

We checked again and no order_conf doesnt work on bankoverschrijving.
It sends its own email that works but it also should send order_conf and its also missing(image1). 
Tried to find why its the case, looks like mail sent is success all the time but the email is not received.
We can also see that some core changes have been done, core changes should have been done with overrides but its done in core so its really hard to find what was changed (image2). 
Order_conf email should have table with prices (image3). But the email you get from bankoverschrijving looks like this: (image4)
Tried to disable mollie modules, remove hooks and so on but doesn’t look like it has anything to do with this issue. If you delete mollie and still can’t get order_conf email with bankoverschrijving then the issue is somewhere else. And if developers want to change how prestashop core works they should create overrides so its easy to find what was changed. Now its impossible to know what was changed but i can see that some parts are changed.
Order 

![image](https://user-images.githubusercontent.com/75423142/162898992-8dadad97-87d7-4472-b6f8-78a847b02104.png)
![image](https://user-images.githubusercontent.com/75423142/162899021-5a7d22d3-e32d-427d-a23c-92916c06865b.png)
![image](https://user-images.githubusercontent.com/75423142/162899058-64b8353c-3784-4832-aa57-437035c40d3b.png)
![image](https://user-images.githubusercontent.com/75423142/162899180-aa52cb80-352a-47e0-b9ce-ae0b0b8fbf21.png)
